templates/apache-guacamole/0/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '2'
+services:
+  database:
+    image: mariadb
+    environment:
+      MYSQL_DATABASE: guacamole_db
+      MYSQL_ONETIME_PASSWORD: 'true'
+      MYSQL_PASSWORD: guacamole_password
```

Comment:
This will be interpreted as a static value for mysql password and mysql user, you should use ${variable} format to reference the environment variable.